packages/ui-extensions/src/surfaces/checkout/targets.ts

```diff
@@ -652,7 +652,13 @@ export interface RenderExtensionTargets {
    */
   'purchase.thank-you.announcement.render': RenderExtension<
     OrderConfirmationApi &
-      StandardApi<'purchase.thank-you.announcement.render'>,
+      StandardApi<'purchase.thank-you.announcement.render'> & {
+        announcement: {
```

Comment:
can we extract this into a type/interface and use it here? Like we do it for others? Like `CartLineItemApi` https://github.com/Shopify/ui-extensions/blob/9c8ecec8c142ed937e768cd40437822183dd1e98/packages/ui-extensions/src/surfaces/checkout/targets.ts#L230